static/app/views/replays/replayTable.tsx

```diff
@@ -234,6 +241,7 @@ function ReplayTableRow({
               pathname: `/organizations/${organization.slug}/replays/${project?.slug}:${replay.id}/`,
               query: {
                 referrer,
+                ...eventView.generateQueryStringObject(),
```

Comment:
Will this also preserve pagination-related params?

Reply:
good thinking ,looks like cursor is not in there. I'll marshal it manually.

Reply:
@ryan953 (I think not having cursor in there is preferable fwiw), it will be hard to maintain the user's position in the list as new Replays roll in

Reply:
ah, true. ok we're good to go then!